tests/Unit/Message/SQSMessageTest.php

```diff
@@ -11,12 +11,12 @@ class SQSMessageTest extends UnitTestCase
 {
     private const RECEIPT_HANDLE = 'this-is-the-receipt-handle';
 
-    /** @var string[] */
+    /** @var array<string>  */
```

Comment:
small thing, probably keep this as `string[]` as it's type-hinted like this through the codebase, but not major